src/main/java/org/jenkinsci/maven/plugins/hpi/hosting/PreferLTS.java

```diff
@@ -0,0 +1,33 @@
+package org.jenkinsci.maven.plugins.hpi.hosting;
+
+import org.apache.maven.project.MavenProject;
+import org.jenkinsci.maven.plugins.hpi.hosting.VerificationMessage.Severity;
+
+public class PreferLTS implements VerificationRule {
+	@Override
+	public VerificationMessage validate(MavenProject project) {
+		MavenProject jenkinsParent = findJenkinsParentPom(project);
+		if (jenkinsParent == null) {
+			return new VerificationMessage(
+					"Your plugin doesn't seem to have the Jenkins parent pom in its POM hierarchy, "
+							+ "it's generally not a normal situation",
+					Severity.WARNING);
+		}
+		if (jenkinsParent.getVersion().split("\\.").length != 3) {
+			return new VerificationMessage("Your plugin does not seem to have a LTS Jenkins release, in general ",
+					Severity.INFO);
+		}
+		return null;
+	}
+
+	private MavenProject findJenkinsParentPom(MavenProject project) {
+		if (!("org.jenkins-ci.plugins".equals(project.getGroupId()) && "plugin".equals(project.getArtifactId()))) {
```

Comment:
Absolutely, adding an IT to catch the good case, 'cause actually the current IT was working because it was always wrong.